Content/src/SAFE.App.Client/package.default.json

```diff
@@ -0,0 +1,28 @@
+{
+    "private": true,
+    "scripts": {
+        "start": "webpack-dev-server",
+        "build": "webpack -p"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.6.4",
+        "@babel/plugin-transform-runtime": "^7.6.2",
+        "@babel/preset-env": "^7.9.6",
+        "@babel/runtime": "^7.6.3",
+        "babel-loader": "^8.0.5",
+        "copy-webpack-plugin": "^5.1.1",
+        "core-js": "^3.3.2",
```

Comment:
I once saw warnings about `core-js` versions - did you encounter that as well? Should we double check it?